src/core/telemetry-config.ts

```diff
@@ -0,0 +1,61 @@
+import { existsSync, mkdirSync, readFileSync, writeFileSync } from 'node:fs'
+import { homedir } from 'node:os'
+import { join } from 'node:path'
```

Comment:
this will break all browser consumers. We can't do `node:` imports in `src/core`.. `src/core` must remain isomorphic.